docs/fundamentals/accessibility.md

```diff
@@ -0,0 +1,395 @@
+
+## Enabling screen readers
+
+Each platform has a different default screen reader to narrate accessibility values:
+
+- Android has TalkBack. For information on enabling TalkBack, see [Enable TalkBack](#enable-talkback).
+- iOS and macOS have VoiceOver. For information on enabling VoiceOver, see [Enable Narrator](#enable-voiceover).
+- Windows has Narrator. For information on enabling Narrator, see [Enable Narrator](#enable-narrator).
+
+### Enable TalkBack
+
+TalkBack is the primary screen reader used on Android. How it's enabled depends on the device manufacturer, Android version, and TalkBack version. However, TalkBack can typically be enabled on your Android device via the device settings:
+
+1. Open the **Settings** app.
+1. Select **Accessibility** > **TalkBack**.
+1. Turn **Use TalkBack** on.
+1. Select **OK**.
+
+> [!NOTE]
+> While these steps apply to most devices, you might experience some differences.
+
+A TalkBack tutorial opens automatically the first time you enable TalkBack.
+
+For alternative methods of enabling TalkBack, see [Turn Talkback on or off](https://support.google.com/accessibility/android/answer/6007100?hl=en-GB&ref_topic=10601570).
+
+### Enable VoiceOver
+
+VoiceOver is the primary screen reader used on iOS and macOS. On iOS, VoiceOver can be enabled as follows:
+
+1. Open the **Settings** app.
+1. Select **Accessibility** > **TalkBack**.
+1. Turn **VoiceOver** on.
+1. Select **OK**.
+
+A VoiceOver tutorial can be opened by selecting **VoiceOver Practice**, once VoiceOver is enabled.
+
+For alternative methods of enabling VoiceOver, see [Turn on and practice VoiceOver on iPhone](https://support.apple.com/guide/iphone/iph3e2e415f/ios) and [Turn on and practice VoiceOver on iPad](https://support.apple.com/guide/ipad/ipad9a246898/ipados).
+
+On macOS, VoiceOver can be enabled as follows:
+
+1. Open the **System Preferences**.
+1. Select **Accessibility** > **VoiceOver**.
+1. Select **Enable VoiceOver**.
+1. Select **Use VoiceOver**.
+
+A VoiceOver tutorial can be opened by selecting **Open VoiceOver Training...**.
+
+For alternative methods of enabling VoiceOver, see [Turn VoiceOver on or off on Mac](https://support.apple.com/guide/voiceover/vo2682/mac).
+
+### Enable Narrator
+
+Narrator is the primary screen reader used on Windows. Narrator can be enabled by pressing the **Windows logo key** + **Ctrl** + **Enter** together. These keys can be pressed again to stop Narrator.
+
+For more information about Narrator, see [Complete guide to Narrator](https://support.microsoft.com/windows/complete-guide-to-narrator-e4397a0d-ef4f-b386-d8ae-c172f109bdb1).
+
+## Accessibility checklist
+
+Follow these tips to ensure that your .NET MAUI apps are accessible to the widest audience possible:
+
+<!-- markdownlint-disable MD032 -->
+
+> [!div class="checklist"]
+> - Make the user interface self-describing. Tag all the elements of your user interface with descriptive text and hints.
```

Comment:
We only want them to use the APIs when they need to (i.e. all text-based controls should be accessible without using any semantic apis)
```suggestion
> - Make sure the user interface is self-describing. Test that all the elements of your user interface are screen reader accessible. Add descriptive text and hints when necessary.
```